### ISSUE TITLE
Updates for Laravel 7 and 8?

### ISSUE THREAD

**Issue by odinns** [beyondcode/laravel-visual-diff#8]
Hello @mpociot 

Thank you for this package 😊 

Would you be interested in a PR to update it to Laravel 7 and 8?

I am working on a project which will use it on 7 and 8, and I have a fork working with Laravel 7 right now, probably also working with Laravel 8.

https://github.com/dwarfhq/laravel-visual-diff/tree/laravel-8

Odinn

**Comment by Jamesking56** [comment]
Yes please I would be interested in using this in a Laravel 8 project

**Comment by odinns** [comment]
@Jamesking56 my fork is working with Laravel 8, you are welcome to use it 😊

**Comment by Jamesking56** [comment]
@odinns Is your fork available on Packagist?

@mpociot Any chance of an update for this?

**Comment by odinns** [comment]
@Jamesking56 I haven't checked, but I pull it into my composer in the usual way.
